index.d.ts

```diff
@@ -95,6 +95,16 @@ export type LinuxResult = {
 
 export type WindowsResult = {
 	platform: 'windows';
+
+	/**
+	Content position and size without Nonclient Area
```

Comment:
It's not obvious what "non client area" means. Spell it out for the user.

Reply:
`Content position and size without Nonclient Area i.e. without the title bar, menu bar, and window frame`

Is this ok?

Reply:
How about:

> Window content position and size, which excludes the title bar, menu bar, and frame.

Reply:
Sounds great. pushed.